src/test/java/com/google/sps/UserTripServletTest.java

```diff
@@ -74,94 +89,167 @@ public void tearDown() {
     helper.tearDown();
   }
 
+  /**
+   * Add a User Entity and no Trip Entity objects to Datastore, and return the 
```

Comment:
what is a "no Trip Entity"?

Reply:
Oh that just means that I'm not adding Trip Entity objects to Datastore. I'll clarify this in the comment.

Reply:
This is not applicable anymore, as I deleted this helper method.